### PR TITLE
Prefer optional chaining over &&

### PR DESCRIPTION
**Pull request by felixfbecker**

### CODE REVIEW

File: web/src/regression/org.test.ts [hunk 1]
```diff
@@ -159,7 +159,7 @@ describe('Organizations regression test suite', () => {
 
             {
                 const quicklinks = await getQuickLinks()
-                if (quicklinks && quicklinks.some(l => l.name === quicklink.name && l.url === quicklink.url)) {
+                if (quicklinks?.some(l=>l.name===quicklink.name&&l.url===quicklink.url)) {
```
Comment (lguychard) [inline web/src/regression/org.test.ts]: Did prettier remove all the whitespace here? That's odd...

File: web/src/enterprise/campaigns/detail/changesets/ChangesetNode.tsx [hunk 2]
```diff
@@ -102,18 +102,7 @@ export const ChangesetNode: React.FunctionComponent<ChangesetNodeProps> = ({
...
-                            isLightTheme={isLightTheme}
-                            node={fileDiffNode}
-                            lineNumbers={true}
-                            location={location}
-                            history={history}
-                            persistLines={node.__typename === 'ExternalChangeset'}
-                            key={i}
-                        ></FileDiffNode>
-                    ))}
+                {fileDiffNodes?.map((fileDiffNode,i)=>(<FileDiffNodeisLightTheme={isLightTheme}node={fileDiffNode}lineNumbers={true}location={location}history={history}persistLines={node.__typename==='ExternalChangeset'}key={i}></FileDiffNode>))}
```
Comment (eseliger) [inline web/src/enterprise/campaigns/detail/changesets/ChangesetNode.tsx]: error

File: web/src/search/results/SearchResultsFilterBars.tsx [hunk 3]
```diff
@@ -35,18 +35,7 @@ export const SearchResultsFilterBars: React.FunctionComponent<{
...
-                            .map((filter, i) => (
-                                <FilterChip
-                                    query={navbarSearchQuery}
-                                    onFilterChosen={onFilterClick}
-                                    key={filter.name + filter.value}
-                                    value={filter.value}
-                                    name={filter.name}
-                                />
-                            ))}
+                    {extensionFilters?.filter(filter=>filter.value!=='').map((filter,i)=>(<FilterChipquery={navbarSearchQuery}onFilterChosen={onFilterClick}key={filter.name+filter.value}value={filter.value}name={filter.name}/>))}
```
Comment (lguychard) [inline web/src/search/results/SearchResultsFilterBars.tsx]: This is also significantly less readable to me than the expanded version 😞

File: web/src/regression/org.test.ts (hunk 1 above)
Comment (felixfbecker) [reply web/src/regression/org.test.ts]: I forgot to run prettier

File: web/src/enterprise/campaigns/detail/changesets/ChangesetNode.tsx (hunk 2 above)
Comment (felixfbecker) [reply web/src/enterprise/campaigns/detail/changesets/ChangesetNode.tsx]: Filed https://github.com/typescript-eslint/typescript-eslint/issues/1266

**Pull request merged**